client/src/partials/cash/cash.html

```diff
@@ -143,6 +143,20 @@
 
                 <hr />
 
+                <div ng-show="CashCtrl.patientBalance < 0" class="form-group">
+                  <div class="alert alert-info">
+                    <span ng-show="CashCtrl.slip.rate">
+                      <strong>{{ 'CASH.CAUTION_COVER' | translate }}</strong> -
+                      <strong>{{ 'FORM.LABELS.SOLD' | translate }}</strong>:
+                      {{ CashCtrl.patientBalance * CashCtrl.slip.rate * -1 | currency:CashCtrl.payment.currency_id }}
```

Comment:
This is complicated logic for a template to be processing. 

Is it possible for this logic to be performed in the controller and then referenced by the template. It could then only be updated when the patient balance or cash slip is changed. 
The only real problem here is that it is difficult to reason about what the template is actually displaying here.